tests/Integration/PaginatorTest.php

```diff
@@ -8,44 +8,48 @@
 
 class PaginatorTest extends TestCase
 {
-    /**
-     * @test
-     */
-    public function it_generates_paginated_data()
-    {
-        $books = [$this->testBooks[0]];
-        $array = $this->fractal
-            ->collection($books, new TestTransformer())
-            ->serializeWith(new JsonApiSerializer())
-            ->paginateWith(new IlluminatePaginatorAdapter(
-                new LengthAwarePaginator($books, 2, 1)
-            ))
-            ->toArray();
+	/**
```

Comment:
This is dude to my PHPstorm setup using tabs instead of spaces. I'll correct this later.